src/guides/v2.3/extension-dev-guide/declarative-schema/db-schema.md

```diff
@@ -154,8 +154,8 @@ A column can have the following attributes:
 
 For more information about each type, refer to the annotations in the corresponding XSD file. The location of the XSD file depends on how you installed Magento.
 
-*  [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc`
-*  [Composer]({{page.baseurl}}/install-gde/composer.html) or [GitHub]({{page.baseurl}}/install-gde/prereq/dev_install.html) installation: `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`
+*  [Archive download]({{page.baseurl}}/install-gde/prereq/zip_install.html) or [Composer]({{page.baseurl}}/install-gde/composer.html): `<Magento_root_directory/vendor/magento/framework/Setup/Declaration/Schema/etc`
```

Comment:
Hi @itfork. Thank you for your contribution. 

With Composer installation of the OpenSource edition, you have the XSD files in  `<Magento_root_directory/lib/internal/Magento/Framework/Setup/Declaration/Schema/etc`. So the current statement is correct.

To double-check this fact you may run the following command in some empty directory:

`composer create-project magento/community-edition`

Reply:
I'm sorry, I didn't know